test/evergreen/doc_update.py

```diff
@@ -29,24 +48,23 @@
 owner = os.getenv("GITHUB_OWNER")
 repo = os.getenv("GITHUB_REPO")
 if (not owner) or (not repo):
-    sys.exit("Please ensure GITHUB_OWNER and GITHUB_REPO environment variables are set.")
+    sys.exit(
+        "Please ensure GITHUB_OWNER and GITHUB_REPO environment variables are set."
+    )
 try:
     installation = app.get_repo_installation(owner, repo)
-    print(f"{installation.id=} for {owner}/{repo}")
 except Exception as e:
     sys.exit(e)
 
 # Get an installation access token
 try:
     installation_auth = app.get_access_token(installation.id)
```

Comment:
This scoping seems a bit weird with the `get_access_token` call now sitting in a try-catch by itself.
I know we discussed this last time but could we merge this block and the block below that calls `git push`?

Reply:
Fair enough. Will update as suggested.

Reply:
See [e504420](https://github.com/wiredtiger/wiredtiger/pull/9854/commits/e504420c0c0808a7c41d10fde2ccac39fc3b97e6)